examples/sorting/Bubble Sort/Go/main.go

```diff
@@ -0,0 +1,38 @@
+// Bubble sort
+//
+// Time complexity: O(N^2)
+//
+// Bubble sort "bubbles" up the array by comparing each array value with the one
+// to the right of it.
+
+package main
+
+import "fmt"
+
+func bubbleSort(arr []int) []int {
+	length := len(arr)
```

Comment:
This seems to be an 8-space tab. Can we narrow this down to be consistent with the rest.

Reply:
Just a note, [go defaults to using tabs for indentation](https://stackoverflow.com/questions/19094704/indentation-in-go-tabs-or-spaces)